scapy/tools/UTscapy.py

```diff
@@ -518,9 +521,13 @@ def import_UTscapy_tools(ses):
 
 
 def run_campaign(test_campaign, get_interactive_session, drop_to_interpreter=False, verb=3, ignore_globals=None):  # noqa: E501
+    global scapy_session
```

Comment:
Please don't use global vars.
You could the session to this function

Reply:
Done